components/auth/ForgotPassword/ForgotPassword.tsx

```diff
@@ -14,20 +14,19 @@ const ForgotPassword = (props: IForgotPasswordProps) => {
   return (
     <div className={style.mainContainer}>
       <div className={style.form_wrapper}>
-        {(isSucceed || isLoading) && (
+        {isLoading && (
```

Comment:
Change "isLoading" to "(isSucceed || isLoading)", please

Reply:
if I do this, then we have constant loader when an email was sent